code/modules/clothing/under/miscellaneous.dm

```diff
@@ -728,3 +728,8 @@
 	icon_state = "hostanclothes"
 	item_state = "hostanclothes"
 	item_color = "hostanclothes"
+
+/obj/item/clothing/under/lawer/blacksuit/iaa

```

Comment:
As I told you before, this should be in code/modules/clothing/under/jobs/security.dm

Same as before. Correct code, wrong location.